memestra/utils.py

```diff
@@ -7,8 +7,8 @@ def resolve_module(module_name, importer_path=()):
     module_path = module_name + ".py"
     bases = sys.path
     if importer_path:
-        bases = chain(os.path.abspath(
-            os.path.dirname(importer_path)), sys.path)
+        bases = chain([os.path.abspath(
```

Comment:
Because this wasn't a list -before- the first entry of this list was a string of chars so instead of `[smth]` it was `['s','m','t','h']`.
This fixes it, would you like me to create a new PR for it or is okay?

Reply:
that's fine.